src/index.css

```diff
@@ -10,3 +10,21 @@ h1 {
   margin: 0;
   margin-bottom: 1em;
 }
+
+.card {
+  background-color: white;
+  padding: 2%;
+  display: inline-block;
+  width: 28%;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-right: 2%;
+}
+
+.card.last {
```

Comment:
I guess we can use [`:last-child`](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child) instead of `.last`.